less/lists.less

```diff
@@ -111,7 +111,7 @@
     margin: auto;
     height: 100%;
     width: 100%;
-    font-size: 0.84em;
+    font-size: 0.90em;
```

Comment:
Could be written `0.9em` instead :wink: 
So you decided not to change the other `font-size: 90%` value? It's not that important.

We could also use rems for solving nested sizing, but this would rather be a global rework of CSS.